README.md

```diff
@@ -103,3 +103,69 @@ To upgrade pinned versions, use the `--upgrade` flag with `pip-compile`.
 Versions can be restricted from updating within the `pyproject.toml` using standard python package version specifiers, i.e. `"black<23"` or `"pip-tools!=6.12.2"`
 
 [`pip-tools`]: https://pip-tools.readthedocs.io/en/latest/
+
+## Working with NPM
+
+This project includes an NPM-based setup for managing front-end assets like styles, scripts, and other resources. The `package.json` file contains predefined scripts to help with building and managing assets.
```

Comment:
It might be worth including a note here that installing the NPM stuff isn't necessary for developing the Django app locally and developers can use the static files for that.

Reply:
Closed in #122